src/core/unit.c

```diff
@@ -6375,6 +6379,92 @@ Condition *unit_find_failed_condition(Unit *u) {
         return failed_trigger && !has_succeeded_trigger ? failed_trigger : NULL;
 }
 
+int unit_can_live_mount(Unit *u, const char *src, const char *dst, sd_bus_error *error) {
+        assert(u);
+        assert(src);
+        assert(dst);
+        assert(error);
+
+        if (!UNIT_VTABLE(u)->can_live_mount || !UNIT_VTABLE(u)->live_mount) {
```

Comment:
i think can_live_mount should be optional. if not set, consider it identical to "yes", i.e. the unit type wants to make no further restrictions if null.

Reply:
i.e. just drop the left side of the || here.